scenarios4/13_The_Way_of_Assassins.cfg

```diff
@@ -192,9 +192,53 @@ A good plan. Let us go."
         [objectives]
             side=1
             [objective]
-                description= _ "Kill Lord Revan, Lord Roth and Lord Kartyn and get to the castle on the eastern border of the map"
+                description= _ "Kill Lord Revan"
                 condition=win
+                [show_if]
+                    [have_unit]
+                        id=lord1
+                    [/have_unit]
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Kill Lord Roth"
+                condition=win
+                [show_if]
+                    [have_unit]
+                        id=lord2
+                    [/have_unit]
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Kill Lord Kartyn"
+                condition=win
+                [show_if]
+                    [have_unit]
+                        id=lord3
+                    [/have_unit]
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "And then get Efraim to the castle on the eastern border of the map"
```

Comment:
The _and then_ connection is not needed here.

Reply:
It's more accurate that way.  The other objectives can be completed in any order, but this one must come last.

Reply:
Aha, okay.